src/go/rpk/pkg/cli/cloud/byoc/install.go

```diff
@@ -70,11 +70,24 @@ func loginAndEnsurePluginVersion(ctx context.Context, fs afero.Fs, cfg *config.C
 		return "", "", false, fmt.Errorf("unable to determine managed plugin path: %w", err)
 	}
 	overrides := cfg.DevOverrides()
-	token, err = oauth.LoadFlow(ctx, fs, cfg, auth0.NewClient(overrides))
-	if err != nil {
-		return "", "", false, fmt.Errorf("unable to load the cloud token: %w", err)
+	if overrides.CloudToken != "" {
+		token = overrides.CloudToken
```

Comment:
This probably needs to be everywhere we create an oauth client in the future, but it's good for now in this PR